source/connect/stable-api.txt

```diff
@@ -0,0 +1,119 @@
+.. _php-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: compatible, backwards, upgrade
+
+.. note::
+
+   The {+stable-api+} feature requires {+mdb-server+} 5.0 or later.
+
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. When you update either your library or server version, 
+the API version changes, which can change the way these operations behave.
+Using the {+stable-api+} ensures consistent responses from the server and 
+provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``MongoDB\Driver\ServerApi`` object and pass the {+stable-api+}
+   version you want to use. Currently, the library supports only version 1.
+#. Construct a ``MongoDB\Client`` object. For the ``driverOptions`` parameter, pass an
+   array that contains the ``serverApi`` connection option. Set this option to the
```

Comment:
```suggestion
   array that contains the ``serverApi`` option. Set this option to the
```

To avoid confusion, I wouldn't refer to this as a "connection option". When constructing a Client (or Manager in the extension directly), we refer to either URI options (specified in either the connection string or second array argument) or driver options (the third array param).